src/main/java/com/arpnetworking/tsdcore/sinks/PeriodicStatisticsSink.java

```diff
@@ -401,11 +435,30 @@ protected Builder self() {
             return this;
         }
 
+        private boolean validate(final ImmutableMap<String, String> ignored) {
```

Comment:
`ignored` corresponds to `_mappedDimensions` in this case right?

Can you just rename the parameter to `mappedDimensions` and use that?